phpstan.neon.dist

```diff
@@ -104,6 +104,12 @@ parameters:
 		-
 			message: '#Cannot assign new offset to string\.#'
 			path: src/JsonSchema/SchemaFactory.php
+		-
+			message: '#Instanceof between bool\|float\|int\|null and ArrayObject will always evaluate to false\.#'
```

Comment:
Can we add inline `@phpstan-ignore-line` comments in the file itself instead of using this file?